.circleci/config.yml

```
@@ -185,6 +191,31 @@ jobs:
                 go get google.golang.org/grpc@v1.2.0
                 go test -v ./contrib/google.golang.org/grpc.v12/...
 
+  copy_artifact_s3:
+    <<: *plain-go113
+    steps:
+      - checkout
+      - run:
+          name: 's3 upload'
+          command: |
+            go run s3index.go
+
+  copy_to_s3:
```

```suggestion
  copy-to-s3:
```
To match `go1.12-build`, or let's change all to underscore if you think that's better.